build/relsignDSCModules.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/bash
+
+# Script to sign DSC modules and drop off signed packages (in zip format) into build share.
+# It generates a checksum file for each DSC module, uploads to azure storage account using SAS_SOURCE_URI,
+# then downloads the signed *.asc file from SAS_DEST_URI and 
+# serializes it into the zip package for given DSC module.
```

Comment:
What's the usage here? You never say how the script should be run and what, if any, parameters should be passed in.

You also never say how this is intended to be called. Is this called directly from Jenkins? Is it embedded elsewhere?